modules/ROOT/nav.adoc

```diff
@@ -17,28 +17,37 @@
+ ** xref:dynamic-expression-field.adoc[Fill in Cards with Values that Change at Runtime]
+ ** xref:custom-expression-field.adoc[Create a DataWeave Expression in a Field]
+ ** xref:fd-add-core-component.adoc[Add Logic, Data Transformations, Error Catching, and More to Flows]
+  *** xref:reference-flow-task-design-center.adoc[Reference One Flow from Another with Flowref]
+  *** xref:error-handling-task-design-center.adoc[Handle Errors with Try]
+ ** xref:fd-transform.adoc[Mapping Output Data to Input Data]
+  *** xref:fd-mapping-examples.adoc[Examples of Mapping with the Transform Card]
+  *** xref:fd-tour-transform-card.adoc[Tour of the Transform Card]
+  *** xref:get-mapping-recommendations.adoc[Get a Recommendation for Mapping Payloads for a Transformation]
+ ** xref:fd-store-data-top.adoc[Store and Reuse Data within a Flow]
+  *** xref:to-create-and-populate-a-variable.adoc[Store All or Part of the Output of a Card in a Variable]
+  *** xref:fd-store-reuse-output.adoc[Store the Entire Output of a Card in a Variable]
  ** xref:manage-dependency-versions-design-center.adoc[Update Dependencies in a Mule-Application Project]
- ** xref:promote-app-prod-env-design-center.adoc[Deploy an Application to an Environment]
- ** xref:to-create-a-mule-application-project.adoc[Clone a Mule App Project]
- ** xref:import-mule-app-project.adoc[Import a Mule Application]
- ** xref:export-studio-design-center.adoc[Export a Project]
- ** xref:viewing-problems-fd-canvas.adoc[List and Navigate to Errors that are in a Mule-App Project]
- ** xref:logger-task-design-center.adoc[Log Data]
- ** xref:view-clear-logs-task.adoc[View the Log for a Mule App Project in Flow Designer]
- ** xref:jump-runtime-manager-task.adoc[Manage a Mule App in Runtime Manager]
- ** xref:reference-flow-task-design-center.adoc[Reference One Flow from Another]
- ** xref:error-handling-task-design-center.adoc[Handle Errors]
- ** xref:input-output-structure-transformation-design-center-task.adoc[Transform Utility Reference]
- ** xref:design-advanced-features.adoc[Advanced Features for Designing Mule Applications]
-  *** xref:to-create-and-populate-a-variable.adoc[About Creating Variables]
-  *** xref:flow-datatype-task.adoc[Set Flow Data Types and Variables]
-  *** xref:change-target-output-transformation-design-center-task.adoc[Change the Target Output of a Transformation]
- ** xref:about-data-types.adoc[About Data Types]
- ** xref:troubleshooting-reference.adoc[Troubleshooting]
+ ** xref:fd-share-to-studio.adoc[Collaborate with Someone who Works in Studio]
+ ** xref:jump-runtime-manager-task.adoc[View Runtime Information about Applications You are Creating]
+ ** xref:fd-troubleshooting.adoc[Troubleshoot in Flow Designer]
+  *** xref:logger-task-design-center.adoc[Log Flow Information at Runtime with Logger]
+  *** xref:troubleshooting-reference.adoc[Error Messages for Flow Designer]
+  *** xref:viewing-problems-fd-canvas.adoc[List and Navigate to Errors that are in a Mule-App Project]
```

Comment:
Remove - (dash) in title. Also could be short _ List and Navigate to Errors in a Mule App Project_

Reply:
Actually, the hyphen is necessary when two nouns together are being used as a single adjective.

Reply:
Ok, there were other Mule App Project without dash that's why I assumed that it should be removed. Makes sense.

Reply:
Thanks. I've found them and fixed them.